exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/logs/LogMarshaler.java

```diff
@@ -42,7 +56,14 @@ final class LogMarshaler extends MarshalerWithSize {
 
   static LogMarshaler create(LogRecordData logRecordData) {
     KeyValueMarshaler[] attributeMarshalers =
-        KeyValueMarshaler.createForAttributes(logRecordData.getAttributes());
+        INCUBATOR_AVAILABLE
+            ? IncubatingUtil.createdExtendedAttributesMarhsalers(logRecordData)
+            : KeyValueMarshaler.createForAttributes(logRecordData.getAttributes());
+
+    int attributeSize =
+        INCUBATOR_AVAILABLE
+            ? IncubatingUtil.extendedAttributesSize(logRecordData)
+            : logRecordData.getAttributes().size();
```

Comment:
Jumping through hoops so serialization logic forks based on whether `opentelemetry-api-incubator` (and ExtendedAttributes) is / isn't available.